web3/_utils/abi.py

```diff
@@ -564,40 +618,40 @@ def get_constructor_abi(contract_abi):
 )
 
 
-def is_recognized_type(abi_type):
+def is_recognized_type(abi_type: TypeStr) -> bool:
     return bool(re.match(TYPE_REGEX, abi_type))
 
 
-def is_bool_type(abi_type):
+def is_bool_type(abi_type: TypeStr) -> bool:
     return abi_type == 'bool'
 
 
-def is_uint_type(abi_type):
+def is_uint_type(abi_type: TypeStr) -> bool:
     return abi_type in UINT_TYPES
 
 
-def is_int_type(abi_type):
+def is_int_type(abi_type: TypeStr) -> bool:
     return abi_type in INT_TYPES
 
 
-def is_address_type(abi_type):
+def is_address_type(abi_type: TypeStr) -> bool:
     return abi_type == 'address'
 
 
-def is_bytes_type(abi_type):
+def is_bytes_type(abi_type: TypeStr) -> bool:
     return abi_type in BYTES_TYPES + ['bytes']
 
 
-def is_string_type(abi_type):
+def is_string_type(abi_type: TypeStr) -> bool:
     return abi_type == 'string'
 
 
 @curry
-def is_length(target_length, value):
+def is_length(target_length: int, value: Any) -> bool:
```

Comment:
[`collections.abc.Sized`](https://docs.python.org/3/library/collections.abc.html#collections.abc.Sized) might be more appropriate for the `value: Any`